intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -238,6 +238,29 @@ \subsection{When will my job start?}
 It might be beneficial to request less resources (e.g. not requesting all cores
 in a workernode), since the scheduler often finds a ``gap'' to fit the job into
 more easily.
+
+\subsection{Specifying the cluster on which to run}
+
+To switch between clusters you can swap the ``cluster'' modules.
```

Comment:
To use other clusters, you can swap...

Also, wrap this in a separate `\ifgent`, to highlight that this section is UGent-specific too, and to avoid problems if this section gets moved (try to ensure the `\ifgent ... \fi` will be moved along with it).